src/main/java/uk/gov/hmcts/cmc/claimstore/services/staff/models/CCJContent.java

```diff
@@ -26,18 +26,21 @@ public CCJContent(Claim claim) {
         requireNonNull(claim);
 
         this.claimReferenceNumber = claim.getReferenceNumber();
-        TheirDetails defendant = claim.getClaimData().getDefendant();
+        CountyCourtJudgment countyCourtJudgment = claim.getCountyCourtJudgment();
+        TheirDetails defendant = countyCourtJudgment.getDefendant();
         this.defendantName = defendant.getName();
         this.defendantAddress = defendant.getAddress();
-        this.paymentRepaymentOption = claim.getCountyCourtJudgment().getPaymentOption().name();
+        this.paymentRepaymentOption = countyCourtJudgment.getPaymentOption().name();
         this.defendantEmail = defendant.getEmail().orElse(null);
-        this.amountToPayByDefendant = formatMoney(
-            ((AmountBreakDown) claim.getClaimData().getAmount()).getTotalAmount()
+        countyCourtJudgment.getRepaymentPlan().ifPresent( repaymentPlan -> {
+                this.amountToPayByDefendant = formatMoney(repaymentPlan.getRemainingAmount());
+            }
         );
+
         this.claimantName = claim.getClaimData().getClaimant().getName();
         this.requestedDate = Formatting.formatDate(claim.getCountyCourtJudgmentRequestedAt());
 
-        claim.getClaimData().getStatementOfTruth().ifPresent(sot -> {
+        countyCourtJudgment.getStatementOfTruth().ifPresent(sot -> {
```

Comment:
Can you please rename `sot` -> `statementOfTruth`? Abbreviations should be avoided if possible.